docs/tasks.md

```diff
@@ -6,173 +6,194 @@ hide_title: false
 
 A list of reusable Cumulus tasks. [Add your own.](adding-a-task.md)
 
+> **NOTE:** For a detailed description of each task, visit the task's `README.md`. Information on the  input or output of a task is specified in the task's `schemas` directory.
+
 ## Tasks
 
 ### [@cumulus/add-missing-file-checksums](https://github.com/nasa/cumulus/tree/master/tasks/add-missing-file-checksums#readme)
 
 Add checksums to files in S3 which don't have one
 
+- Readme: Check out the [README](https://github.com/nasa/cumulus/tree/master/tasks/add-missing-file-checksums#readme) for additional information.
```

Comment:
Definitely open to changing this if you can think of a better alternative

Reply:
What's the context for this change? Not totally following.

Reply:
Yeah, the context for this change is to make task information clearer for users to find. This was something I could limit to sync-granule, but thought I would update this entire task.md file. Someone from the MAAP team brought up that READMEs weren't something they knew they could find in our repo, and suggested that the docs be updated to clarify that

Reply:
Got it. Makes sense to me!